src/components/TextInput/TextInput.vue

```diff
@@ -8,6 +8,23 @@
       :sr-only-label="srOnlyLabel"
       :tooltip-content="tooltipContent"
     />
+    <div
+      data-testId="copiedTip"
+      :class="[
+        'z-10 absolute w-20 p-2 text-xs rounded-md bg-gray-700 text-white',
+        'transform translate-x-20 -translate-y-9 duration-300',
+        {'opacity-100 -translate-y-11': copied },
+        {'opacity-0': !copied }
+      ]"
+    >
+      <div class="flex">
+        <Check class="h-4 w-4" />
+        <div class="ml-1.5">
+          Copied
+        </div>
+      </div>
+      <div class="absolute bg-transparent w-0 h-0 m-auto border-l-8 border-r-8 border-t-8 border-l-transparent border-r-transparent border-t-gray-700 -bottom-2 left-0 right-0" />
+    </div>
```

Comment:
I'm curious why you went the approach of changing the opacity based on `copied` rather than using a `v-if="copied"`.  I think leaving this in the DOM and just visually hiding it could be a little confusing for assistive technology devices which will always "see"(read) it.

I wonder if instead you could do something like:
```
<div aria-role="alert" aria-live="polite">
  <div
      data-testId="copiedTip"
      v-if="copied"
      :class="z-10 absolute w-20 p-2 text-xs rounded-md bg-gray-700 text-white transform translate-x-20 -translate-y-9 duration-300 opacity-100 -translate-y-11"
    >
      <div class="flex">
        <Check class="h-4 w-4" />
        <div class="ml-1.5">
          Copied
        </div>
      </div>
      <div class="absolute bg-transparent w-0 h-0 m-auto border-l-8 border-r-8 border-t-8 border-l-transparent border-r-transparent border-t-gray-700 -bottom-2 left-0 right-0" />
    </div>
</div>
```

This way it's inside an `aria-role="alert"` which will make sure it's read by a screen reader as soon as its shown but it's only in the DOM when shown/read.

Reply:
I believe she went with opacity for the transition animation?

Reply:
Yes, we are dealing with opacity and not display for the animation to work. 
The v-if destroys the element too fast, so the animation would not work, the element would disappear.
I am not familiar with anything aria..Would the aria role work without the v-if?

Reply:
There is a way to also destroy the element though, just a bit more JS and another timeout and we can make v-if work too - I think

Reply:
Hmm not quite.  When using aria "live regions" (`aria-live` and/or `aria-role="alert"`) you need the element that will contain the content that needs to be announced to the user to be in the DOM ahead of the changes being made within it in order for the assistive technologies to recognize the change and be able to announce it (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).  So you want that outer `<div>` to always be in the DOM.  What's inside it can come and go though.

I'm not an expert on animations but it seems like it has to be easier to accommodate an animation inside a dynamically added element... I thought I remembered seeing some examples in a course I took recently.  Let me see if I can find it...

Reply:
Yep here it is: https://vuejs.org/v2/guide/transitions.html.  So you can wrap the element with the `v-if` with a `<transition>` and then you can add the type of transition with the `name` property and add styling to its specific classes if it needs tweaking.

Reply:
aaah I'd forgotten about the <transition> tag indeed, let's try this

Reply:
I think you still need to remove the `v-if="withCopyButton"` on the outer `<div>` and then it'll be 🤩

Reply:
The alert div only needs to be there when the `withCopyButton` prop is true, otherwise it is a plain textinput with no copy function. no? 🤔